### DISCUSSION TITLE
Nuxt and Esbuild

### DISCUSSION THREAD

**Discussion by eulier1** [nuxt/nuxt#9076] [Ideas]
Hey.

Since this Front-end tooling https://github.com/vitejs/vite it's using esbuild to pre-bundle dependency 10-100X faster than JS bundler-based.

Is nuxt team, looking a way to replace or provide esbuild support?.

**Comment by danielroe** [comment]
@eulier1 You might be interested in https://github.com/nuxt/vite or https://github.com/galvez/nuxt-esbuild-module.

**Reply by eulier1** [reply]
Thank you, very much, have a nice day!
